src/OperationalInsights/OperationalInsights.Test/ScenarioTests/TableTests.ps1

```diff
@@ -14,46 +13,143 @@
 
 <#
 .SYNOPSIS
-Test Table CRUD
+Test List all tables for a given WS and get table on a table that does not exist
 #>
 function Test-TableCRUD
 {
 	# setup
-	$rgName = Get-ResourceGroupName
-	$workspaceName = Get-ResourceName
-	$loc = Get-ProviderLocation
 	$tableNotFound = Get-ResourceName
-	$initialRetention = 60
-	$updattedRetention = 61
-
 	$rgNameExisting = "dabenham-dev"
 	$wsNameExisting = "dabenham-eus"
 	$tableNameExisting = "dabenhamDev_CL"
 
-
 	try
 	{
-		# create new RG for the test
-		New-AzResourceGroup -Name $rgname -Location $loc
-
-		# create new WS for the test
-		$workspace = New-AzOperationalInsightsWorkspace -ResourceGroupName $rgname -Name $workspaceName -Location $loc
-
 		# get all existing tables
 		$allTable = Get-AzOperationalInsightsTable -ResourceGroupName $rgName -WorkspaceName $workspaceName
 		Assert-NotNull $allTable
-		Assert-True {$allTable.Count -gt 0}
+		Assert-True {$allTable.Count -gt 1}
 		
 		# get table that does not exist 
 		Assert-ThrowsContains {Get-AzOperationalInsightsTable -ResourceGroupName $rgName -WorkspaceName $workspaceName -tableName $tableNotFound} 'NotFound'
+	}
+	finally
+	{
+		# Cleanup
```

Comment:
do you want to do something in this finally block?

Reply:
removed